src/config/load_config_utils.c

```diff
@@ -0,0 +1,74 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   load_config_utils.c                                :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: syan <syan@student.42.fr>                  +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2024/04/05 14:28:03 by syan              #+#    #+#             */
+/*   Updated: 2024/04/05 14:29:51 by syan             ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "cub3d.h"
+
+void	check_digit(char **rgb, char **element)
+{
+	if (!rgb[0] || !rgb[1] || !rgb[2])
+	{
+		ft_free_split(&rgb);
+		ft_free_split(&element);
+		exit_error(EMRGB);
+	}
+	if ((ft_atoi(rgb[0]) == 0 && ft_strcmp(rgb[0], "0") != 0) \
+			|| (ft_atoi(rgb[1]) == 0 && ft_strcmp(rgb[1], "0") != 0) \
+				|| (ft_atoi(rgb[2]) == 0 && ft_strcmp(rgb[2], "0") != 0))
+	{
+		ft_free_split(&rgb);
+		ft_free_split(&element);
+		exit_error(EMRGB);
+	}
+}
+
+void	check_cf(char **element, t_cub3d *word)
+{
+	char	**rgb;
+
+	check_mult_id(element, word);
+	rgb = check_rgb(element);
+	if (ft_strcmp(element[0], "C") == 0)
+		word->graphics.ceiling_color = ft_rgb_to_int(ft_atoi(rgb[0]), \
+			ft_atoi(rgb[1]), ft_atoi(rgb[2]));
+	else if (ft_strcmp(element[0], "F") == 0)
+		word->graphics.floor_color = ft_rgb_to_int(ft_atoi(rgb[0]), \
+			ft_atoi(rgb[1]), ft_atoi(rgb[2]));
```

Comment:
The backslash `\` at the end of a line allows to bypass the norme.

I propose this alternative, that goes more according to the norme and maintain readability and adherence to coding standards.

```C
if (ft_strcmp(element[0], "C") == 0)
	word->graphics.ceiling_color = ft_rgb_to_int(ft_atoi(rgb[0]),
			ft_atoi(rgb[1]), ft_atoi(rgb[2]));
else if (ft_strcmp(element[0], "F") == 0)
	word->graphics.floor_color = ft_rgb_to_int(ft_atoi(rgb[0]),
			ft_atoi(rgb[1]), ft_atoi(rgb[2]));
```